### PR TITLE
for Swift3

### PR DESCRIPTION
**Pull request by 335g**
- @noescape/@escaping
- operator definition

### CODE REVIEW

File: Prelude/Application.swift [hunk 1]
```diff
@@ -2,21 +2,16 @@
 
 // MARK: Operators
 
-infix operator |> {
+precedencegroup Association {
```
Comment (robrix) [inline Prelude/Application.swift]: Would `Application` be a better name?

**Comment by robrix** [comment]
Thanks for the PR! Just the one question :point_up:

File: Prelude/Application.swift (hunk 1 above)
Comment (335g) [inline Prelude/Application.swift]: OK. I'm not so good at English. I fix it 🙇

**Comment by robrix** [comment]
:bow: Thank you so much! This is great.

**Comment by 335g** [comment]
Thanks for the review & merge 🙇

**Pull request merged**